Sources/Core/Microsoft.StreamProcessing/Operators/Beat/PartitionedBeatPipe.cs

```diff
@@ -88,7 +88,7 @@ public override unsafe void OnNext(StreamMessage<TKey, TPayload> batch)
                 long* sourceVOtherPtr = sourceVOther;
                 for (int row = 0; row < count; row++)
                 {
-                    if ((sourceBitVector[row >> 6] & (1L << (row & 0x3f))) == 0 || *sourceVOtherPtr == long.MinValue)
+                    if ((sourceBitVector[row >> 6] & (1L << (row & 0x3f))) == 0 || *sourceVOtherPtr < 0)
```

Comment:
Using appropriate constant for `0` like `DateTimeOffset.MinValue.UtcTicks` might be better

Reply:
That would be misleading since this is not a DateTime.

---
In reply to: [474309811](https://github.com/microsoft/Trill/pull/136#discussion_r474309811) [](ancestors = 474309811)